release-calendar/src/client/components/ChannelTable.tsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as React from 'react';
+import {channels} from '../../types';
+import {Square} from './Square';
+
+export class ChannelTable extends React.Component<{}, {}> {
+  //TODO(ajwhatson):
+  // add event handling with onClick functions
+  // send state from app carrying Map<Channel, boolean> to map selected channels
```

Comment:
Might be cleaner to send an array of selected channels instead of a map where the value is a boolean.